frontend/src/entities/configuration/configuration.tsx

```diff
@@ -57,6 +58,7 @@ export function Configuration () {
             }
         >
             <SpaceBetween direction='vertical' size='xl'>
+                <DynamicConfiguration/>
```

Comment:
Is this supposed to be in or not? Why was it removed in the first revision?

Reply:
Forgot to remove it, undoing. We'll officially add it to the admin page once all the features are done and we're happy with the UI.